### DISCUSSION TITLE
Does the interactive mode with alpaca-lora output a record of prompts and responses somewhere?

### DISCUSSION THREAD

**Discussion by shawnschulz** [ggml-org/llama.cpp#1049] [Q&A]
Hey all,

Was just wondering if the interactive mode using alpaca-lora outputted a record of prompts and responses somewhere for easy viewing, was going to set it to do this anyways but if it's already being saved in json or text file somewhere didn't see the need to do this twice.

**Comment by DannyDaemonic** [comment]
Sorry, nothing is logged or written to disk.
